### PR TITLE
re-enable tubefill.

### PR DESCRIPTION
**Pull request by donhth**
Compiled and tested that it does indeed refill the tube with adamantine.

### CODE REVIEW

**Review by lethosor — COMMENTED** [review]
Can you mention this in docs/changelog.txt? There should be some examples of re-enabled plugins from previous releases.

Also, update docs/plugins/tubefill.rst to remove the `unavailable` tag

**Comment by donhth** [comment]
Note: the changelog doesn't tell the whole truth. It replenishes mined out adamantine and replaces all the mined out special stuff (clusters consisting of: water, magma, obsidian, gemstones) in the tube with adamantine.  

I think this is ok because:

1) Users interested in this tool are most likely only after the adamantine.
2) I believe the tool is clear enough about its intention to fill the tube with adamantine.

If there's community interest in replenishing the tube to its original state (as much as possible) then an option can always be added in for that later.  Better to have something than nothing, for players that want more adamantine now.

File: docs/changelog.txt [hunk 1]
```diff
@@ -67,6 +67,11 @@ Template for new versions:
 
 ## Removed
 
+# 50.10-r2
+
+## New Plugins
+- `tubefill`: (reinstated) replenishes mined-out adamantine
+
```
Comment (myk002) [inline docs/changelog.txt:74]: We'll create the 50.10-r2 header when we tag the release. For now, the `tubefill` entry should go under "New Tools" under the "Future" heading.

**Comment by myk002** [comment]
> Note: the changelog doesn't tell the whole truth. It replenishes mined out adamantine and replaces all the mined out special stuff (clusters consisting of: water, magma, obsidian, gemstones) in the tube with adamantine.

Feel free to add more info to the help file!

I do agree that replacing mined-out gems with adamantine is perfectly fine

**Comment by donhth** [comment]
Alright, maybe I did it correctly this time. I think I get it now. 

I can add more info to the changelog regarding the differences if that's what you were referring to by 'help file.'  Otherwise, I'd prefer to do that in a separate PR, if that's ok.

**Comment by myk002** [comment]
By "help file" I was referring to `tubefill.rst` -- that's what players will see online at https://docs.dfhack.org/en/latest/docs/tools/tubefill.html and in-game in `gui/launcher` when they type `tubefill`.

**Review by myk002 — COMMENTED** [review]
code change looks good, though a note about how mined-out gem-studded walls will be replaced with adamantine in `tubefill.rst` would be welcomed

**Pull request merged**